src/Database/Driver.php

```diff
@@ -361,4 +362,14 @@ public function __debugInfo()
             'connected' => $this->isConnected()
         ];
     }
+
+    /**
+     * Returns the session isolationLevel
+     * Sets it, if a value is given
+     *
+     * @param string $isolationLevel Transaction Isolation Level
+     * @return string
+     * @throws TransactionIsolationLevelNotSupportedException
+     */
+    abstract public function transactionIsolationLevel($isolationLevel = null);
```

Comment:
cannot have it as abstract in a minor version, since it would break other people's code